src/eigenspacedecomposition.jl

```diff
@@ -14,7 +14,7 @@ function row_echelon_form!(A::AbstractMatrix{T}) where {T<:Number}
         end
 
         w = inv(A[pos, i])
-        A[pos, :] .*= w
+        A[pos, i:end] .*= w
```

Comment:
Is this a bugfix ? Was the previous implementation incorrect ?

Reply:
I believe it's small performance optimization: all the columns left of `i` should be already cleared by now. Or so I think the loop below is doing.